app/controllers/candidate_interface/degrees/degree_controller.rb

```diff
@@ -1,6 +1,25 @@
 module CandidateInterface
   module Degrees
     class DegreeController < BaseController
+      def new_university_degree
```

Comment:
This would be my preference as well.

Reply:
Interesting, that was my preference too, I opt in on this to avoid to be different from the current implementation too much. I will move to another controller then :)

Reply:
Done on c045f03515